priv/templates/channel/channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule <%= module %>Channel do
+  use <%= base %>.Web, :channel
+
+  def join("<%= plural %>:lobby", _auth_message, socket) do
+    {:ok, socket}
+  end
+
+  def join("<%= plural %>:" <> _<%= singular %>_id, _auth_message, socket) do
+    {:ok, socket}
+  end
+
+  <%= for event <- events do %>
+    def handle_in("<%= event %>", attrs, socket) do
+      broadcast! socket, event, attrs
```

Comment:
How 'bout if we have channel generators for specific use cases? Like maybe have `mix.phoenix.channel.gen.crud User users Post` where it would generate a `UserChannel` module that joins `users` topic  and handles CRUD events like `create_post`, `delete_post`, `show_post`, `show_posts` and `update_post`. Its create_post handler will look like the sample in the docs:
```elixir
def handle_in("create_post", attrs, socket) do
  changeset = Post.changeset(%Post{}, attrs)

  if changeset.valid? do
    Repo.insert(changeset)
    {:reply, {:ok, changeset}, socket}
  else
    {:reply, {:error, changeset}, socket}
  end
end
```

Not yet sure how the generic channel-generator will create `handle_in` functions for events passed.